lib/SPIRVProducerPass.cpp

```diff
@@ -157,31 +160,26 @@ struct SPIRVOperand {
 private:
   SPIRVOperandType Type;
   std::string LiteralStr;
-  SmallVector<uint32_t, 4> LiteralNum;
+  uint32_t LiteralNum[2];
```

Comment:
Simplified to make copy light-weight, so these can be collected by value vs pointer management.